src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -339,6 +356,7 @@ private static IConfig CreateConfig(CommandLineOptions options, IConfig globalCo
                 config.AddJob(baseJob);
 
             config.AddExporter(options.Exporters.SelectMany(exporter => AvailableExporters[exporter]).ToArray());
+            config.AddExporter(CustomExporters.Select(c => c.Value).ToArray());

```

Comment:
The registered exporters should only be added if the user specified it in the command line.